src/index.scss

```
@@ -18,6 +18,64 @@
+
+#uploadOptionsTable {
+  @media (max-width: 1200px) {
+    .MuiTableCell-alignLeft:not(:first-of-type) {
+      display: none !important;
+    }
+  }
+}
+
+.ag-theme-material {
+
+  $odd-cell-background-color:  #cee5ff;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)